server/endpoints/profile/photo.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const Photo = require('../../models/photo');
+const User = require('../../models/user');
+
+const router = new express.Router();
+
+router.get('/profile/:profile_id/photo/:photo_id', function(req, res) {
+  const photo = req.photo;
+  const currentUser = req.user;
+  Photo.findById(req.params.id)
```

Comment:
req.params.photoId, to su ti ovi parametri u url-u u liniji 7 koji zapocinju s dvotockom.